rover/src/storage.ts

```diff
@@ -0,0 +1,52 @@
+import "reflect-metadata";
```

Comment:
rebase needed?

Reply:
this is actually being used by typeorm https://stackoverflow.com/questions/49618719/why-typeorm-needs-reflect-metadata

Reply:
no I mean this is not a configuration file, is it? (storage.ts)

Reply:
(I thought I saw it in a previous PR)

Reply:
this file was missing, its being added here to ensure that the test don't break when the configure is applied